### ISSUE TITLE
Caught /sync error TypeError: i.default is not a function

### ISSUE THREAD

**Issue by menturion** [matrix-org/matrix-js-sdk#1216]
Since v5.0.0 I'm geting the following issue:

`Caught /sync error TypeError: i.default is not a function` at Object.r.removeHiddenChars.

**Comment by turt2live** [comment]
Do you have the full stack trace?

**Comment by menturion** [comment]
Please find below the stack trace when using the unminified version:

```
Caught /sync error TypeError: unhomoglyph_1.default is not a function
at Object.removeHiddenChars (http://localhost/browser-matrix.js:33)
at _updateDisplayNameCache (http://localhost/browser-matrix.js:50)
at http://localhost/browser-matrix.js:7
at Object.forEach (http://localhost/browser-matrix.js:9)
at RoomState.setStateEvents (http://localhost/browser-matrix.js:9)
at EventTimeline.addEvent (http://localhost/browser-matrix.js:20)
at EventTimelineSet.addEventToTimeline (http://localhost/browser-matrix.js:12)
at EventTimelineSet.addLiveEvent (http://localhost/browser-matrix.js:8)
at Room._addLiveEvent (http://localhost/browser-matrix.js:27)
at Room.addLiveEvents (http://localhost/browser-matrix.js:10)
```

**Comment by menturion** [comment]
This issue is still active in v5.1.0-rc.1.

**Comment by t3chguy** [comment]
It sounds more like an issue with your npm/yarn and how it pulled in unhomoglyph. https://github.com/nodeca/unhomoglyph/blob/master/index.js

**Comment by menturion** [comment]
I am using the "browser-matrix.min.js" file
(https://github.com/matrix-org/matrix-js-sdk/releases/download/v5.1.0-rc.1/browser-matrix.min.js) or the non-minified version.

**Comment by menturion** [comment]
This issue is still active in v5.1.1-rc.1 so that the browser JS assets don't work.

**Comment by drale2k** [comment]
Running into the same issue with Version 6. I was able to fix it like described here https://github.com/matrix-org/matrix-js-sdk/issues/1371

**Comment by t3chguy** [comment]
This should be fixed as of `6.1.0-rc.1` https://github.com/matrix-org/matrix-js-sdk/releases/download/v6.1.0-rc.1/browser-matrix.js

**Comment by t3chguy** [comment]
Reopen if you see it from then onwards.
